Apps/Sandcastle/gallery/MSAA.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta name="description" content="Add MSAA to the scene." />
+    <meta name="cesium-sandcastle-labels" content="Post Processing" />
```

Comment:
I think the sandcastle should go in a different tab since it doesn't use post processing, though I get why it was added here. `Showcases` would be fine.